ch03.asciidoc

```diff
@@ -386,18 +386,47 @@ This includes which public payment channels are connecting nodes, which capacity
 ==== Onion routing
 
 If the sending node of a payment has selected a path that is supposed to be used to make the payment, the Lightning Network uses an onion routing scheme similar to the famous TOR-network.
-The routing scheme is called the SPHINX mixformat and will be explained in detail in a later chapter.
-For now we want to focus on its properties for the transport of payments which we also call onions.
-
-1. The most important property is that a routing node can only see on which channels it received an onion and on which channel to set up an HTLCs and thus to which peer to forward the onion. This means that no routing node can know who initiated the payment and for whom the payment is supposed to be. The exception of course would be if the node is the recipient. In that case it would know that it was the final destination.
-2. The onions are small enough to fit into a single TCP/IP package and actually even a link layer frame. This will make traffic analysis for intruding the privacy of the payments almost impossible.
-3. The onions are constructed in a way that they will always have the same length independent of the position of the processing node along the path.
-4. Onions can have up to 20 hops included allowing for sufficiently long paths.
-5. The encryption of the onion for every hop uses different ephemeral encryption keys with every single onion. Should a key (in particular the private key of the public node key) leak at some point in time an attacker who collected onions cannot decrypt the other onions that have been stored.
+The routing scheme is called the "SPHINX mixformat" and will be explained in detail in a later chapter.
+Note that Lightning Onion routing via "SPHINX mixformat" is only similar to the TOR network routing in conecpt, but as a protocol as well as in implementation it is definitely different from the TOR network.
+For now we want to focus on its properties for the transport of payments.
+A payment package used for routing is called an "onion".
+footnote:[The term "onion" was originally used by the TOR project.
+The acronym TOR stands for "The Onion Router".
+Moreover, the Tor network is also called the Onion network. 
+The Tor project uses an onion as its logo and the top level domain name used by TOR services on the internet is ".onion".]
+As a simplified physical analogy imagine the payment package as a sealed box with a onion inside. 
+On its route from payer to payee the sealed box is handed from node to node. 
+Sealed just means that the box is encrypted and only the receiving node can open it.
+As will be explained, each node opens the box (using a cryptographic key), takes out the onion, and peals off the outermost layer of this onion.
```

Comment:
actually the opening and taking out of the onion payload does not require a cryptographic key. it is rather the pealing off that requires this